### ISSUE TITLE
Clickable list item

### ISSUE THREAD

**Issue by oliverjam** [FAC-Sixteen/chip-shop#5]
https://github.com/FAC-Sixteen/chip-shop/blob/81baeb90aff5e80054c3dfb1cf4bd5e72ec34bae/src/components/Letters.js#L55

Generally if you're attaching an `onClick` to something it should be a button. That way assistive technology recognises it as interactive. Also you get some extra behaviours for free—keyboard focus and handling (enter/spacebar will fire a button too).

You can put a button inside an `li`
